src/main/java/codechicken/lib/render/CCRenderEventHandler.java

```diff
@@ -51,4 +53,10 @@ public void onBlockHighlight(RenderHighlightEvent.Block event) {
             RenderUtils.bufferShapeHitBox(mat, event.getMultiBufferSource(), event.getCamera(), voxelHit.shape);
         }
     }
+
+    @OnlyIn (Dist.CLIENT)
+    @SubscribeEvent (priority = EventPriority.LOW)
+    public void screenOpenEvent(ScreenEvent.Opening event) {
+        CursorHelper.resetCursor();
```

Comment:
Whenever _any_ screen opens, we reset the cursor? Surely we should only reset the cursor when _our_ gui is closed?

Reply:
This was mostly just so that I could be absolutely sure we wouldn't accidentally leave a custom cursor active. 
But I can use Screen#removed because it looks like that will always get called.